skbio/util/_misc.py

```diff
@@ -226,7 +229,9 @@ def safe_md5(open_file, block_size=2 ** 20):
     return md5
 
 
-@experimental(as_of="0.4.0")
+@deprecated(as_of="0.4.2-dev", until="0.4.4",
```

Comment:
Let's go with 0.5.1 since 0.5.0 is tentatively the next release. Sorry for giving you the wrong `until` @Kleptobismol